sw/tools/generators/gen_autopilot.ml

```diff
@@ -423,7 +423,7 @@ let write_settings = fun xml_file out_set ap ->
   *)
 let gen_autopilot main_freq xml_file h_dir out_set =
   try
-    let ap_xml = Xml.parse_file xml_file in
+    let ap_xml = ExtXml.parse_file xml_file in
```

Comment:
does it make sense to keep duplicating the error handling here instead of using ExtXml.parse_file?